### PR TITLE
[author][new] Add billboard.js w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by netil**
Pull request for issue: #11409
Related issue(s): 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/naver/billboard.js
 * Official website (optional, not the repository): https://naver.github.io/billboard.js/
 * NPM package url (optional): https://www.npmjs.com/package/billboard.js
 * License and its reference: MIT, https://github.com/naver/billboard.js/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 31
   - Count of stars: 908
   - Count of forks: 1,151
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/billboard.js/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "billboard.js",
+  "description": "Re-usable easy interface JavaScript chart library, based on D3 v4+",
+  "author": "NAVER Corp.",
+  "filename": "billboard.min.js",
+  "homepage": "http://naver.github.io/billboard.js/",
```
Comment (dakshshah96) [inline ajax/libs/billboard.js/package.json]: Please use `https` here.

File: ajax/libs/billboard.js/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "graph",
+    "svg"
+  ],
+  "license": "MIT",
+  "npmName": "billboard.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.+(map))"
```
Comment (dakshshah96) [inline ajax/libs/billboard.js/package.json]: I don't think we need to exclude the `map` files. Seems fine to use `**/*` inside `dist`.

**Comment by netil** [comment]
@dakshshah96 thanks. I did requested changes (sorry not squashing, I did changes directly from web)

**Comment by netil** [comment]
@sufuf3, would you mind to squash?

**Comment by sufuf3** [comment]
@netil Done 😄 
Help squash, rebase to master, modified line 26 from `  ]  ` to `  ]` (delete two space which in the end of this line).

**Comment by sufuf3** [comment]
@cdnjs/intern3 @dakshshah96 
Please help review this PR, Thank you.

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help review this PR, Thank you.

**Pull request merged**